pointcloud/basic_octree_test.go

```diff
@@ -678,23 +672,10 @@ func TestCachedMaxProbability(t *testing.T) {
 			children: nil,
 			nodeType: leafNodeFilled,
 			point:    &PointAndData{P: r3.Vector{1, 2, 3}, D: d},
-			maxVal:   emptyProb,
+			maxVal:   defaultConfidenceThreshold,
 		}
 		test.That(t, node, test.ShouldResemble, filledNode)
 	})
-
-	t.Run("setting negative values", func(t *testing.T) {
```

Comment:
why is it safe to remove this test.

Reply:
Don't quite remember why I removed it.  but I don't think it needed to exist in the first place.

Reply:
Yes I did that when I initially pushed this code